cmd/tapcli/universe.go

```diff
@@ -8,7 +8,6 @@ import (
 	tap "github.com/lightninglabs/taproot-assets"
 	"github.com/lightninglabs/taproot-assets/proof"
 	"github.com/lightninglabs/taproot-assets/taprpc"
-	"github.com/lightninglabs/taproot-assets/taprpc/universerpc"
```

Comment:
Weird that linter didn't complain with both instances...

Reply:
Yeah updated it to follow the abbrevation we use elsewhere in `cmd`.